src/briefcase/platforms/android/gradle.py

```diff
@@ -182,13 +233,38 @@ def add_options(self, parser):
             " or an AVD name ('@emulatorName') ",
             required=False,
         )
+        parser.add_argument(
+            "-X",
+            action="append",
+            dest="extra_emulator_args",
+            help="Additional arguments to use when starting the emulator",
+            required=False,
+        )
+        parser.add_argument(
+            "--shutdown-on-exit",
+            dest="shutdown_on_exit",
```

Comment:
Replacement of `-` with `_` is done automatically, so there's no need to specify `dest`.